style.css

```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+.logo {
+  margin-top: 20px;
+  text-decoration: none;
+  padding: 10px 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 25px;
+  color: #28352f;
+}
+
+.hamburger {
+  margin-top: 10px;
+  margin-right: 10px;
+  width: 20px;
+  height: 20px;
+}
+
+.intro {
+  background-image: url(/media/header-illsutration-mobile@2x.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+  position: relative;
+  padding-top: 15px;
+}
+
+.categories ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+  padding-bottom: 20px;
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.categories li {
+  margin: 0.5rem;
+  background-color: #e5e5e5;
+  padding: 8px 12px;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+}
+
+.list ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+  padding-bottom: 20px;
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.list li {
+  margin: 0.5rem;
+  background-color: #fff;
+  padding: 8px 12px;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+  color: #36b37e;
+}
+
+.socialmedia ul {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  list-style: none;
+  padding-top: 110px;
+  margin-bottom: 50px;
+}
+
+.socialmedia ul li {
+  text-decoration: none;
+}
+
+.intro h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  margin: 0;
+  padding-top: 50px;
+}
+
+.intro h2 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #36b37e;
+  margin-top: 5px;
+}
+
+.intro p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  padding-top: 40px;
+}
+
+.arrow {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  padding-top: 20px;
+}
+
+.works {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 28px;
+  text-align: center;
+  color: #172b4d;
+  margin: 0;
+  padding-bottom: 40px;
+}
+
+.cards {
+  display: grid;
+  justify-content: center;
+  padding: 10px 10px 10px 10px;
+  grid-template-columns: 330px;
+}
```

Comment:
Well done for following the figma design quite closely. It is [required](https://github.com/microverseinc/curriculum-html-css/blob/e1fd98988a32ecaf3767a9832e6cea0ff0d18792/portfolio/2_mobile.md?plain=1#L51) to have the container holding all projects have a grid layout while the individual projects have a display: flex. You have the container for all projects to be `.works` while each project is enclosed in `.cards`

Reply:
@emmiiorji thanks for your feedback. But it seems like I confused you by giving different names to my classes. So the main container class in my project here is ".cards" and cards itself is a ".card" class. 
I used the grid  ".cards" class and flex in ".card" class respectively as per the requirement you mentioned above.

Reply:
Hi @Ferizad,

I see, you're quite correct. I have approved accordingly.